libs/model/src/config.ts

```diff
@@ -400,5 +406,9 @@ export const config = configure(
           'SKALE_PRIVATE_KEY must be set to a non-default value in production.',
         ),
     }),
+    IMAGE_GENERATION: z.object({
+      USE_RUNWARE: z.boolean().optional(),
```

Comment:
We have a pattern for flags -> `FLAG_USE_RUNWARE`. Additionally, you should refine the `IMAGE_GENERATION` object such that an error is thrown if `FLAG_USE_RUNWARE` is true but `RUNWARE_API_KEY` is not set.